storage/reads/aggregate_resultset.go

```diff
@@ -22,28 +22,36 @@ type windowAggregateResultSet struct {
 	err          error
 }
 
-func NewWindowAggregateResultSet(ctx context.Context, req *datatypes.ReadWindowAggregateRequest, cursor SeriesCursor) (ResultSet, error) {
-	if nAggs := len(req.Aggregate); nAggs != 1 {
-		return nil, errors.Errorf(errors.InternalError, "attempt to create a windowAggregateResultSet with %v aggregate functions", nAggs)
+// IsAscendingWindowAggregate checks two things: If the request passed in
+// is using the `last` aggregate type, and if it doesn't have a window. If both
+// conditions are met, it returns false, otherwise, it returns true.
+func IsAscendingWindowAggregate(req *datatypes.ReadWindowAggregateRequest) bool {
```

Comment:
This is a bad name for the function - IsWindowedAggregateWithLast would be better. But this is a backport so ok